x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/feedback_modal.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiConfirmModal } from '@elastic/eui';
+
+export function FeedbackModal({
+  isFeedbackModalVisiable,
+  setsIsFeedbackModalVisiable,
+  refetch,
+}: {
+  isFeedbackModalVisiable: boolean;
+  setsIsFeedbackModalVisiable: (value: boolean) => void;
+  refetch: () => void;
+}) {
+  const closeModal = () => {
+    setsIsFeedbackModalVisiable(false);
+    refetch();
+  };
```

Comment:
Perhaps the `refetch()` should happen in the `<EntityEnablement />` component, in callback for the `onClose` event? Calling refresh here seems to make this modal too smart